docs/team/limerencee.adoc

```diff
@@ -4,61 +4,60 @@
 :stylesDir: ../stylesheets
 
 == 1. Introduction
-=== 1.1 Purpose
-This portfolio serves as a showcase of my roles and contributions to the team project `**THRIFT**`. This project was part of
-the requirements of a year two software engineering module that I have taken as an undergraduate in NUS.
+This portfolio serves as a showcase of my roles and contributions to the team project THRIFT. This project was part of
```

Comment:
Maybe you could just mention CS2103T briefly, rather than describing SE module without a name? Diana did mention this briefly.

Reply:
Also, consider using "serves to showcase" rather than "serves as a showcase", i think it may flow better grammatically.

Reply:
Fixed, thanks for the tips!